src/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,83 @@
+import "./Profile.css";
+import SideBar from "../SideBar/SideBar";
+import { useState } from "react";
+
+function Profile() {
+  const [searchInput, setSearchInput] = useState("");
+  const [albums, setAlbums] = useState([]);
+
+  const handleChange = (e) => {
+    setSearchInput(e.target.value);
+  };
+
+  const handleSearch = async () => {
+    try {
+      const accessToken = localStorage.getItem("spotify_access_token");
+
+      const response = await fetch(
+        `https://api.spotify.com/v1/search?type=album&q=${encodeURIComponent(
+          searchInput
+        )}`,
+        {
+          headers: {
+            Authorization: `Bearer ${accessToken}`,
+          },
+        }
```

Comment:
Please, define the api functions inside `SpotifyApi.js`, and then import them into the components to separtate the logics